src/components/toolbox/dropdown/dropdown.css

```diff
@@ -0,0 +1,64 @@
+@import '../../app/variables.css';
+
+:root {
+  --dropdown-box-shadow: 0 8px 16px 0 rgba(29, 72, 119, 0.16);
+  --dropdown-list-hover-color: rgba(204, 217, 228, 0.2);
+}
+
+.dropdown ul {
```

Comment:
I think for this would be better to use the pattern:

```
.dropdown {

& ul {...}
& li, .selected {...}
& .values {...}
}
```
so that if ever we replace the name, then it's only in one place, and postcss will output the same.

Reply:
hm well i see your point, but this is not a name we gave, it's part of the react toolbox dropdown. they gave that classname, so we won't have influence on that. do you still want me to change that?